card/lib/card/query.rb

```diff
@@ -61,7 +61,8 @@ class Query
                            link_to linked_to_by
                            include included_by                                ),
       conjunction:     %w(and or all any),
-      special:         %w(found_by not sort match complete extension_type),
+      special:         %w(found_by not sort match name_match complete junction_complete
```

Comment:
`match` does a lot of weird stuff that I don't understand but the thing that I understand in that method is that it matches only complete words